### DISCUSSION TITLE
关于SEVI大作业最终提交文件的问题

### DISCUSSION THREAD

**Discussion by CresDai** [physics-data/faq#118] [Q&A] [answered]
在SEVI大作业中，如果完成了提高任务，应该在一个脚本中完成（当作不同版本），还是分为多个脚本？因为这些任务要求的参数似乎有所不同。 
谢谢！@littlewu2508

**Comment by littlewu2508** [comment, marked as answer]
请使用多个脚本，并在实验报告中说明该如何运行这些脚本、以何种顺序运行。

CresDai ***@***.***> 于2021年9月3日周五 下午8:49写道：
